db/migrate/20210208105018_clean_up_international_addresses_data.rb

```diff
@@ -1,26 +0,0 @@
-class CleanUpInternationalAddressesData < ActiveRecord::Migration[6.0]
-  def change
```

Comment:
@cwrw to check with candidate team around removing this data migration

Reply:
I think let's delete this, it's already run on production so we don't need to keep it anymore

Reply:
as in keep in PR and we'll merge as is